pipeline-scripts/release.groovy

```diff
@@ -45,7 +45,7 @@ def destReleaseTag(String releaseName, String arch) {
  *      Valid advisories must be in QE state and have a live ID so we can
  *      include in release metadata the URL where it will be published.
  */
-Map stageValidation(String quay_url, String dest_release_tag, int advisory = 0, boolean permitPayloadOverwrite = false, boolean permitAnyAdvisoryState = false) {
+Map stageValidation(String quay_url, String dest_release_tag, int advisory = 0, boolean permitPayloadOverwrite = false, boolean permitAnyAdvisoryState = false, String nightly, String arch) {
```

Comment:
... positional parameters added after "optional" parameters? might be tripping up syntax... @joepvd any idea?

Reply:
I reformat the indentation,don't know if it related, please check again.